docs/model_configuration.md

```diff
@@ -501,6 +501,59 @@ the device kind of the instance, for instance, KIND_CPU is "cpu", KIND_MODEL is
   ]
 ```
 
+### Rate Limiter Config
+
+Instance group optionally specifies [rate limiter](rate_limiter.md)
+config which controls how the rate limiter operates on the instances
+in the group. The rate limiter configuration includes the following
+specifications:
+
+#### Resources
+
+The set of resources required to execute a model instance. Essentially,
+a resource is a mapping of a name to its corresponding count.
+By default, the resource counts are local to the device running the
+instance. However, a resource, with `global` flag set to `True`, will
+be shared across the system. Loaded models can not specify a resource
+with the same name both as global and non-global.
+
+#### Priority
+
+Priority serves as a weighting value to be used for prioritizing across
+all the instances of all the models. An instance with priority 2 will be
+given 1/2 the number of scheduling chances as an instance with priority
+1.
+
+The following example specifies the instances in the group requires 
```

Comment:
Say something about how the instance group create 3 model instances (one each on gpus 0 1 and 2). Then you can say explicitly which resources are generates per-device and which are generated globally...